source/cloud/nvidia/brev.md

```diff
@@ -4,41 +4,66 @@ review_priority: "p0"
 
 # NVIDIA Brev
 
-The [NVIDIA Brev](https://brev.dev/) platform provides you with GPU instances on cloud providers including [Amazon Web Services](https://aws.amazon.com/) and [Google Cloud](https://cloud.google.com) with CUDA, Python, Jupyter Lab, all set up.
+The [NVIDIA Brev](https://brev.dev/) platform provides you a one stop menu of available GPU instances across many cloud providers, including [Amazon Web Services](https://aws.amazon.com/) and [Google Cloud](https://cloud.google.com), with CUDA, Python, Jupyter Lab, all set up.
 
-You can get up and running with RAPIDS in a few quicks thanks to the Brev RAPIDS quickstart.
+## Brev Instance Setup 
+There are two ways that you can get up and running with RAPIDS in a few quicks thanks to the Brev RAPIDS quickstart:
```

Comment:
The wording is a bit redundant, like using  quicks and quickstart
```suggestion
There are two ways that you can get up and running with RAPIDS in a few steps, thanks to the Brev RAPIDS quickstart:
```

Reply:
Gah, that was supposed to be `clicks`, but i can commit `steps`